deploy/oci-images.nix

```diff
@@ -0,0 +1,37 @@
+{ inputs }:
+let
+  inherit (inputs) std self nixpkgs;
+  inherit (nixpkgs.lib) removePrefix;
+  inherit (self) operables;
+  inherit (self.sourceInfo) lastModifiedDate;
+
+  mkImage = name: std.lib.ops.mkStandardOCI {
+    name = "registry.ci.iog.io/dapps-world-${name}";
```

Comment:
Should we use the public registry?

Reply:
Yeah we could switch to only using those. I originally kept the internal ones because they were easier to iterate over, but the entrypoints/images have mostly stabilized.